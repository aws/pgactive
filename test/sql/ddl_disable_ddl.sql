ALTER DATABASE regression RESET bdr.skip_ddl_replication;
ALTER DATABASE postgres RESET bdr.skip_ddl_replication;
