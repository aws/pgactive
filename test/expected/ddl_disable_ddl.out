ALTER DATABASE regression RESET pgactive.skip_ddl_replication;
ALTER DATABASE postgres RESET pgactive.skip_ddl_replication;
